src/routes/web/results.ts

```diff
@@ -9,4 +10,23 @@ module.exports = [
     path: webRoutePaths.results,
     handler: SearchResultsController.renderSearchResultsHandler,
   },
+  {
+    method: 'GET',
+    path: webRoutePaths.guidedSearch,
+    handler: SearchResultsController.renderGuidedSearchHandler,
+  },
+  {
+    method: 'POST',
+    path: webRoutePaths.guidedSearch,
+    handler: SearchResultsController.renderGuidedSearchHandler,
+    options: {
+      validate: {
+        payload: dateSchema,
+        failAction: (request, h, error) => {
```

Comment:
<img width="485" alt="image" src="https://github.com/DEFRA/ncea-frontend/assets/131792244/7acb3374-9041-4e10-958b-d732f3fd8feb">

1. Here, multiple validation messages pop up. Are we happy to put them in a single line or should we check with the team about placing them one by one or returning only one message at a time?
2. The HTML name attribute of the Day field appears in the first message.
3. If validation is present, the field border needs to be coloured.